apps/taskman/models.py

```diff
@@ -40,14 +41,30 @@ class JiraResolution(models.TextChoices):
     TEST_PENDING = "Test Pending"
 
 
+WORKFLOW_TO_JIRA_STATUS = {
+    WorkflowModel.OSIMState.NEW: (JiraStatus.NEW, JiraResolution.NONE),
+    WorkflowModel.OSIMState.TRIAGE: (JiraStatus.REFINEMENT, JiraResolution.NONE),
+    WorkflowModel.OSIMState.PRE_SECOND_ASSESSMENT: (
+        JiraStatus.REFINEMENT,
+        JiraResolution.NONE,
+    ),
```

Comment:
This is when the four states are maybe not enough. We cannot distinguish these two states here. We can do so based on other workflow conditions but important is that we will need to distinguish them in Jira too - so we can provide complete workflow boards. It can also be done base on other attributes of course so I would like to just clarify it. How do you plan to distinguish them?